web.py

```diff
@@ -79,27 +71,22 @@
         HOST_CACHE_SIZE cache entries.
     """
 
-    host_config = host_cache.get(hostname)
+    if hostname not in host_cache:
 
-    if not host_config:
+        config = {}
+
+        url = urljoin(FRANKLIN_API_URL, '/v1/domains/')
+        params = {'domain': hostname, 'key': FRANKLIN_API_KEY}
+        headers = {'User-Agent': 'franklin-server'}
```

Comment:
The endpoint current exists and currently does no have security on it.
But when it does, that will be passed in the header instead of as a query param.